src/main/java/org/SaucedemoTesting/pageobjects/CartPage.java

```diff
@@ -0,0 +1,32 @@
+package org.SaucedemoTesting.pageobjects;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class CartPage {
+    @FindBy(xpath = "//button[@id='add-to-cart-sauce-labs-bolt-t-shirt']")
+    private WebElement btn_addToCart_SLBTSH;
+    @FindBy(xpath = "//button[@id='remove-sauce-labs-bolt-t-shirt']")
+    private WebElement btn_removeFromCart_SLBTSH;
```

Comment:
PageObjects o sa te rog sa le muti in folderul de "test"

Reply:
Ok.